app/models/manageiq/providers/hawkular/datawarehouse_manager.rb

```diff
@@ -46,10 +46,17 @@ def self.raw_connect(hostname, port, token, alerts = false)
     end
 
     def connect(options = {})
-      @client ||= self.class.raw_connect(hostname,
-                                         port,
-                                         authentication_token('default'),
-                                         options[:alerts])
+      @clients ||= {}
+      @clients[options[:alerts] ? :alerts : :metrics] ||= self.class.raw_connect(
```

Comment:
The variable would be used, just very locally:
```
client_type = options[:alerts] ? :alerts : :metrics
@clients[client_type] || = ...
```

But that highlights the fact there are approximately four representations of client type around here:
1. `::Hawkular::Alerts::AlertsClient` vs `::Hawkular::Metrics::Client`
2. :alerts vs :metrics here
3. raw_connect takes positional boolean [3.5. connect takes named boolean]
  Q: can any more endpoints become needed later?  E.g. Openshift manager `fetch_hawk_inv()` creates aggregated client then does `.strings.get_data()`.
4. calling `alerts_client` (and maybe `metrics_client`) instead of `connect`.

Is it feasible to change `raw_connect` and `connect` interface to take symbol (:alerts vs :metrics) instead of booleans?
Then you have less representations, can do simpler `options[:type] || :metrics`, and might not need `{alerts,metrics}_client` aliases...